ui/components/ui/sender-to-recipient/sender-to-recipient.component.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import copyToClipboard from 'copy-to-clipboard';
 import { NameType } from '@metamask/name-controller';
+import { toUnicode } from 'punycode';
```

Comment:
`punycode` lib I think is deprecated, we should avoid using it

Reply:
I don't think we should rely on any other package before removing the `toASCII` util from `ENSController` - these utils match each other since both using same encoding from `punycode` package

Reply:
In that case, will be nice to mention that as a comment here.

Reply:
Good call, added a comment.